### ISSUE TITLE
for last two (or more) versions sparkle update check results in `Update Error!`

### ISSUE THREAD

**Issue by foss-** [ospfranco/sol#100]
Unsure when this started but can say that for at least the last two versions the sparkle update check results in `Update Error!` and the update does not happen.

```
2023-01-08 09:39:11.339031+0100 0xe8384    Error       0x13e7d8             24283  0    sol: (Sparkle) [org.sparkle-project.Sparkle:Sparkle] Error: An error occurred while downloading the update. Please try again later. (null) (URL https://raw.githubusercontent.com/ospfranco/sol/main/releases/Sol.app.zip)
2023-01-08 09:39:11.339075+0100 0xe8384    Error       0x13e7d8             24283  0    sol: (Sparkle) [org.sparkle-project.Sparkle:Sparkle] Error: A network error occurred while downloading the update. not found (404) (null) (URL (null))
```

Since `https://raw.githubusercontent.com/ospfranco/sol/main/releases/Sol.app.zip` is throwing a 404 it seems that location has been changed?

**Comment by ospfranco** [comment]
Hmm it seems the AppCast.xml file is not correctly generated... I modified the latest release, would you mind giving it another try?

**Comment by foss-** [comment]
Tried updating 1.1.123 to 1.1.124. The reported problem is solved. The download starts but on extract we now see another error:

Update Error!
The update is improperly signed and could not be validated.

**Comment by ospfranco** [comment]
just released 1.1.25, give that one a try please

**Comment by foss-** [comment]
You mean try updating to 1.1.25 or install 1.1.25 and then do update check? Checking from 1.1.23 still shows `The update is improperly signed and could not be validated.`

**Comment by ospfranco** [comment]
I meant updating, but installing should also work

**Comment by foss-** [comment]
Saw 1.1.28 is now available but updating from 1.1.23 still fails with above error.

**Comment by ospfranco** [comment]
Ok, found the issue, it's because I changed machines and the signature is not there. Unfortunately this means the updater will be broken from 1.1.20 to future versions.

Please download the latest version 1.1.30 manually and it should work starting there.
